cap-terraform/.gitignore

```diff
@@ -0,0 +1,8 @@
+**/creds/*
+.terraform
+*.backup
+*.tfstate
+scf*/
```

Comment:
Where are files matching this path prefix created?

Reply:
thats from parent directory: https://github.com/SUSE/cap-terraform/blob/master/.gitignore
I won't touch that as Satadru experimented with that. If that gets cleaned in cap-terraform I will remove it from here